### PR TITLE
Add Voqs to Virtual Switch

### PR DESCRIPTION
**Pull request by skbarista**
Add Voqs to Virtual Switch. This is needed for virtual chassis tests to pass it https://github.com/Azure/sonic-swss/pull/1913

### CODE REVIEW

File: vslib/SwitchStateBase.h [hunk 1]
```diff
@@ -108,6 +108,8 @@ namespace saivs
                     _In_ uint32_t sys_port_count,
                     _In_ const sai_system_port_config_t *sys_port_cfg_list);
 
+            sai_status_t create_voqs();
+            sai_status_t create_voq_per_sysport(_In_ sai_object_id_t sys_port_id);
```
Comment (kcudnik) [inline vslib/SwitchStateBase.h:112]: move paremater to second line like in other functions

Comment (skbarista) [reply vslib/SwitchStateBase.h]: I fixed the parameter. The coverage seems to be not detecting the coverage added my new test. I checked in my local environment, the new functions are exercised by the test I added. @kcudnik

**Comment by kcudnik** [comment]
please satisfy code coverage

**Comment by kcudnik** [comment]
not sure about that, i believe i had the same issue that tests added in perl script was not counting coverage, maybe they generate gcov files in different location or something like that, and it's not picked up by code coverage engine, i don't have root cause here :(

**Review by kcudnik — APPROVED** [review]

**Comment by yxieca** [comment]
@skbarista this change cannot be cherry-picked cleanly to 202205. Can you enter new PR for 202205 branch?

**Comment by skbarista** [comment]
@yxieca I raised https://github.com/sonic-net/sonic-sairedis/pull/1161

**Comment by yxieca** [comment]
https://github.com/sonic-net/sonic-sairedis/pull/1162 is the 202205 PR.

**Pull request merged**